### PR TITLE
adding screenshot of lift&shift status

### PR DESCRIPTION
**Pull request by emuus**
Although the story does not include info about migration details per say, as suggested by mail by CC, I have created a section (placeholder) for this info as part of this story. This section can be filled (with screenshot) at later time.

### CODE REVIEW

File: end-user/en/source/pages/migration/automatic-migration.rst [hunk 1]
```diff
@@ -44,3 +44,18 @@ In order to run a migration:
 	11. Click ``next`` to view the migration status.
 	12. You can return to the ``Lift & Shift`` page to view the status of your migrations.
 
+	.. image:: /images/migration-status.png
+
+	From the status page you can view more detailed information about the appliance scanned by clicking: 
+
+		* ``Details`` for more information about the appliances, including the cloud ID, cloud account used for publication, etc.
```
Comment (lqueiroga) [inline end-user/en/source/pages/migration/automatic-migration.rst]: this details link in fact is the details about the publication. The cloud ID appears only when publish succeed

File: end-user/en/source/pages/migration/automatic-migration.rst [hunk 2]
```diff
@@ -44,3 +44,18 @@ In order to run a migration:
...
 	12. You can return to the ``Lift & Shift`` page to view the status of your migrations.
 
+	.. image:: /images/migration-status.png
+
+	From the status page you can view more detailed information about the appliance scanned by clicking: 
+
+		* ``Details`` for more information about the appliances, including the cloud ID, cloud account used for publication, etc. 
+		* ``View Scan Details`` which will bring you to the details of the scan appliance on the ``Re-platform`` page. From here you can see all the packages and files that are part of the appliance.
+		* ``View Generations Details`` which will bring you to the ``Generations`` page. This page lists ALL generations (not just from this migration). From here you can launch a publish.
+		* ``View Publication Details`` which will bring you to the ``Generations`` page.
```
Comment (lqueiroga) [inline end-user/en/source/pages/migration/automatic-migration.rst]: ``Publications`` page.
I know that in fact it's the same page than the Generation one.

File: end-user/en/source/pages/migration/automatic-migration.rst [hunk 3]
```diff
@@ -44,3 +44,18 @@ In order to run a migration:
 	11. Click ``next`` to view the migration status.
 	12. You can return to the ``Lift & Shift`` page to view the status of your migrations.
 
+	.. image:: /images/migration-status.png
```
Comment (lqueiroga) [inline end-user/en/source/pages/migration/automatic-migration.rst]: All this must be under the paragraph below "Viewing Details of an Automatic Migration" ?

File: end-user/en/source/pages/migration/automatic-migration.rst [hunk 4]
```diff
@@ -44,3 +44,18 @@ In order to run a migration:
...
+		* ``Details`` for more information about the appliances, including the cloud ID, cloud account used for publication, etc. 
+		* ``View Scan Details`` which will bring you to the details of the scan appliance on the ``Re-platform`` page. From here you can see all the packages and files that are part of the appliance.
+		* ``View Generations Details`` which will bring you to the ``Generations`` page. This page lists ALL generations (not just from this migration). From here you can launch a publish.
+		* ``View Publication Details`` which will bring you to the ``Generations`` page.
+
+Viewing Details of an Automatic Migration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once your automatic migration has completed, you can view the details of the status and details of the machine migrated from the ``Lift & Shift`` tab on the ``Migrations`` page.
+
```
Comment (lqueiroga) [inline end-user/en/source/pages/migration/automatic-migration.rst]: See my comment above, I think that all you added above must be below it....

File: end-user/en/source/pages/migration/automatic-migration.rst (hunk 2 above)
Comment (emuus) [reply end-user/en/source/pages/migration/automatic-migration.rst]: Yes but for the user, the page name is the one that appears at the top, so here "Generation"

**Pull request merged**